packages/start/src/runtime/server-handler.ts

```diff
@@ -78,19 +80,28 @@ async function handleServerFunction(h3Event: HTTPEvent) {
   const instance = request.headers.get("X-Server-Instance");
   const singleFlight = request.headers.has("X-Single-Flight");
   const url = new URL(request.url);
-  let filepath: string | undefined | null, name: string | undefined | null;
+  let functionId: string | undefined | null, name: string | undefined | null;
   if (serverReference) {
     invariant(typeof serverReference === "string", "Invalid server function");
-    [filepath, name] = serverReference.split("#");
+    [functionId, name] = serverReference.split("#");
   } else {
-    filepath = url.searchParams.get("id");
+    functionId = url.searchParams.get("id");
     name = url.searchParams.get("name");
-    if (!filepath || !name) throw new Error("Invalid request");
+    if (!functionId || !name) throw new Error("Invalid request");
+  }
+
+  const serverFnInfo = serverFnManifest[functionId];
+  let fnModule: undefined | { [key: string]: any };
+
+  if (process.env.NODE_ENV === "development") {
```

Comment:
Why do we have to load the server function via vinxi during development 🤔, does `serverFnInfo.importer` not work in dev?

Reply:
I'm not entirely sure. I kinda copied this over from Tanstack. If you want I can see if its not needed

Reply:
It seems like its needed cause otherwise it errors

Reply:
Oh I guess this really must be some limitation then, there is a comment about this in the tanstack sources:
https://github.com/TanStack/router/blob/266f5cc863cd1a99809d1af2669e58b6b6db9a67/packages/start-server-functions-handler/src/index.tsx#L81

Since this is not mentioned in the README of the tanstack server functions plugin, I recommend adding a comment referencing the original tanstack code, so that we quickly can compare the integration at a later time 🤔